packages/Checkbox/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [0.2.0] - 2020-01-14
+### Added
```

Comment:
The fact that you've `Added` the new `Checkbox.states` doesn't create the break though, it's the fact that you've `Removed` the `checkboxStates` export, right? Both should be included in the changelog, but it's the thing that breaks the component that I'd be most concerned with (followed closely by how to fix it).